scripts/install-jq.js

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+'use strict'
+
+var which = require('which')
+var wget = require('download')
+var fs = require('fs')
+var path = require('path')
+
+var JQ_NAME = 'jq'
+
+var JQ_RELEASE_INFO = {
```

Comment:
Should we extract this into some config file, maybe [package.json](https://docs.npmjs.com/misc/config#per-package-config-settings)?